recipes-bsp/firmware-imx/imx-boot-firmware-files_8.18.bb

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2018-2020 NXP
+SUMMARY = "Freescale i.MX firmware for 8 family"
+DESCRIPTION = "Freescale i.MX firmware for 8 family"
+
+require firmware-imx-${PV}.inc
+
+inherit deploy nopackages
+
+do_install[noexec] = "1"
+
+DEPLOY_FOR                  = ""
+DEPLOY_FOR:mx8-generic-bsp  = "mx8"
+DEPLOY_FOR:mx8m-generic-bsp = "mx8 mx8m"
```

Comment:
Looks like this PR mistakenly adds 8 family Cadence firmware to 8M family.

Reply:
Fixed in #1532.